vision/classification/extact_imagenet.py

```diff
@@ -81,4 +81,4 @@ def main():
     extract_val(val_tar_fname, os.path.join(target_dir, 'val'))
 
 if __name__ == '__main__':
-    main()
+    main()
```

Comment:
looks like the line ending got changed

Reply:
I tried overwriting it with the original file but it shows that there is no difference between the original and my revision.